common/test/db_fixtures/xblock_configuration.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
Note to reviewers: this JSON configuration populates the XBlockConfiguration model for bok choy tests. Previously the state was set in bok_choy.py as a settings variable was being used.